basic-extra-rhel/bootc/etc/opentelemetry-collector/configs/10-exporter.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Not sure if we want such a default exporter config. We can also use the agent to ship the config.